core/web/router.go

```diff
@@ -292,6 +292,10 @@ func v2Routes(app chainlink.Application, r *gin.RouterGroup) {
 		authv2.POST("/keys/csa/import", auth.RequiresAdminRole(csakc.Import))
 		authv2.POST("/keys/csa/export/:ID", auth.RequiresAdminRole(csakc.Export))
 
+		ethKeysGroup := authv2.Group("", auth.Authenticate(app.SessionORM(),
+			auth.AuthenticateByToken,
+			auth.AuthenticateBySession,
+		))
```

Comment:
Should this apply to the legacy 'eth' routes too?
If not, can we move it down to ~308?

Reply:
Originally planned on that, but the improved API response could be a small push to move towards the new 'evm' routes instead. Moved to 308 :)